CMakeLists.txt

```diff
@@ -224,7 +224,7 @@ if(ENABLE_QT)
     endif()
 
     foreach(M ${QT5_REQUIRED_MODULES})
-        find_package(Qt5${M} ${QT5_MINIMUM} QUIET)
+        find_package(Qt5${M} ${QT5_MINIMUM} REQUIRED)
```

Comment:
This probably breaks `-DENABLE_QT=AUTO`, where we want to build Qt client if possible, and not building it otherwise is fine too. Not sure how useful this mode is (along with other options supporting `AUTO`), if only but CI, but if not then support for it needs to be removed as well.